pre_commit/commands/run.py

```diff
@@ -13,6 +13,7 @@
 from typing import Sequence
 from typing import Set
 from typing import Tuple
+from unicodedata import east_asian_width
```

Comment:
I tend to prefer `import unicodedata` and then `unicodedata.east_asian_width` for stdlib modules